firstapp/executeCuj

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+PRE_CUJ=${1?ERROR: no Pre CUJ given}
+POST_CUJ=${2?ERROR: no Recorded CUJ given}
+NUM=${3:-1}
+RECORD=${4:-"n"}
+
+
+
+adb logcat -c
+sed -i -e "s/int NUM_ITERATIONS = .*;/int NUM_ITERATIONS = $NUM;/g" ./app/src/androidTest/java/com/example/firstapp/UserCujTest.java
+sed -i -e "s/static String.*preCUJ = .*;/static String[] preCUJ = $PRE_CUJ;/g" ./app/src/androidTest/java/com/example/firstapp/UserCujTest.java
+sed -i -e "s/static String.*postCUJ = .*;/static String[] postCUJ = $POST_CUJ;/g" ./app/src/androidTest/java/com/example/firstapp/UserCujTest.java
+
+if [ "$RECORD" = "r" ] 
+then 
+	sed -i -e "s/long RECORDING_BUF = .*;/long RECORDING_BUF = 1000;/g" ./app/src/androidTest/java/com/example/firstapp/UserCujTest.java
```

Comment:
This isn't the right way to pass data into the test.  Alternatives:
1. passing in a file
2. adb shell am instrument -e ...
3. ./gradlew argument passing